src/versionUtils.ts

```diff
@@ -96,11 +96,11 @@ export async function pingGithubUrl(urlStr: string): Promise<boolean> {
 }
 
 export function isOldTagNaming(tagName: string): boolean {
-	return tagName.startsWith('0.') || tagName.startsWith('1.0.');
+	return tagName?.startsWith('0.') || tagName?.startsWith('1.0.');
```

Comment:
if tagName can be undefined, shouldn't be better to declare it in the parameter as such?

Reply:
I just double checked the calling functions and it seems we will not have that case. Anyways having this additional check will not hurt.

Reply:
There's a difference between saying a parameter is optional (i.e. `(tagName? : string)`) and checking the value of the parameter passed in. But maybe it could be a bit more precise with `(tagName : string | undefined)` and a check below for a more easily spotted result if the incoming value is undefined? 

That said, I think the way Lars has it now gets the result we're looking for (i.e. false) for undefined values without all the extra work.